modules/ROOT/pages/apikit-4-for-odatav2.adoc

```diff
@@ -0,0 +1,17 @@
+= APIkit for OData V2
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+APIkit for OData v2 is a framework designed to simplify the development of RESTful APIs that adhere to the OData v2 specification. OData, also known as the Open Data Protocol, is a protocol that enables the creation and consumption of RESTful APIs. It allows data querying and manipulation through standard HTTP requests.
+
+APIkit for OData v2 was developed as an extension of APIkit for REST. It builds upon its foundation and utilizes RAML APIs. This integration between APIkit for OData v2 and APIkit for REST combines their functionalities and benefits.
```

Comment:
```suggestion
APIkit for OData v2 is an extension of APIkit for REST. It builds upon its foundation and uses RAML APIs. This integration between APIkit for OData v2 and APIkit for REST combines their functionalities and benefits.
```

Reply:
Having two "it"s in this sentence makes the meaning unclear:

It builds upon its foundation and uses RAML APIs.

Maybe: It builds upon the foundation of APIkit for REST...

Reply:
I like it! Thanks!